recipes/onnx-caffe2/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "onnx-caffe2" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "8ecd11a76d074a355734ebe3c44a93b4bc4e3f5d75fc7330d56498a83aa79a88" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script:
+    - python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - pytest-runner
+    - enum34  # [py2k]
+  run:
+    - python
+    - onnx
+    # NB: caffe2 is not conda managed, so don't put it as a "dep"
```

Comment:
Are there any plans to put this on conda-forge?

Reply:
Yes, but it's a bit complicated and upstream keeps breaking the build, so it might take some time before it gets settled.

Reply:
I know that feeling, good luck!